components/VoteList/VoteListItem.tsx

```diff
@@ -452,7 +452,7 @@ export function VoteListItem({
 }
 
 function VoteText({ voteText }: { voteText: string | undefined }) {
-  if (!voteText) return <LoadingSkeleton width="8vw" />;
+  if (voteText === undefined) return <LoadingSkeleton width="8vw" />;
```

Comment:
I realized that this would have shown a permanent loader when passed an empty string because of the falsy check